src/Network/Http/Response.php

```diff
@@ -388,7 +388,7 @@ protected function _getJson()
             return $this->_json;
         }
         $data = json_decode($this->_body, true);
-        if ($data) {
+        if (is_array($data)) {
```

Comment:
> Maybe allow everything except null?

@dereuromark  In that case the body and return value gets set to null which isn't a problem either since that's the default value.